lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -27,6 +27,15 @@
   }, 1500);
 </script>
 
+<object id="5934a"></object>
+<object id="5934b"></object>
+<script>
+  document.getElementById("5934a").matches = "See #5934";
```

Comment:
can you add a comment on why these lines are here so we aren't mystified at some point in the future :)

Reply:
yup, done